RoaringBitmap/src/main/java/org/roaringbitmap/longlong/HighLowContainer.java

```diff
@@ -180,6 +180,8 @@ public void serialize(ByteBuffer buffer) throws IOException {
         : buffer.slice().order(LITTLE_ENDIAN);
     if (art.isEmpty()) {
       byteBuffer.put(EMPTY_TAG);
+      byteBuffer.position();
```

Comment:
Is this line useless here?

Reply:
Correct. It was unnecessary and it has been removed. Thanks for reporting it.